### PR TITLE
Add PR link to the issue template

### PR DESCRIPTION
**Pull request by kevinbackhouse**
Every bounty submission is connected to a pull request in the codeql or codeql-go repos, so it should be part of the issue template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-slayer.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ assignees: ''
 
 ---
 
+## Query
+
+*Link to pull request with your CodeQL query:*
+
+Relevant PR: https://github.com/github/codeql/pull/nnnn
```
Comment (xcorail) [inline .github/ISSUE_TEMPLATE/bug-slayer.md:14]: Is there always a PR for the bug slayer?

Comment (kevinbackhouse) [reply .github/ISSUE_TEMPLATE/bug-slayer.md:14]: I think so? You have to write a query to get the bounty. Of course, you may have written the query some time ago, so the PR may have already been merged.

Comment (xcorail) [reply .github/ISSUE_TEMPLATE/bug-slayer.md:14]: Yeah, and IIRC the query doesn't need to be a general purpose one, eligible for the real codeql repo ... 
Well, this bug slayer is a mess, we really need to revise it. 
I will approve in the meantime, if there is a PR it's better to have it in a template

File: .github/ISSUE_TEMPLATE/bug-slayer.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ assignees: ''
 
 ---
 
+## Query
+
+*Link to pull request with your CodeQL query:*
```
Comment (xcorail) [inline .github/ISSUE_TEMPLATE/bug-slayer.md:12]:
```suggestion
*OPTIONAL - Link to pull request with your CodeQL query:*
```

Comment (xcorail) [reply .github/ISSUE_TEMPLATE/bug-slayer.md:12]: What about that?

**Review by xcorail — APPROVED** [review]

**Pull request merged**